lib/trento_web/openapi/schema/database.ex

```diff
@@ -0,0 +1,80 @@
+defmodule TrentoWeb.OpenApi.Schema.Database do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  alias TrentoWeb.OpenApi.Schema.{ResourceHealth, Tags}
+
+  defmodule DatabaseInstance do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "DatabaseInstance",
+      description: "A discovered HANA Database Instance on the target infrastructure",
+      type: :object,
+      properties: %{
+        sap_system_id: %Schema{type: :integer, description: "SAP System ID", format: :uuid},
+        sid: %Schema{type: :string, description: "SID"},
+        tenant: %Schema{type: :string, description: "Tenant"},
+        instance_number: %Schema{type: :string, description: "Instance Number"},
+        instance_hostname: %Schema{type: :string, description: "Instance Hostname"},
+        features: %Schema{type: :string, description: "Instance Features"},
+        http_port: %Schema{type: :integer, description: "Instance HTTP Port"},
+        https_port: %Schema{type: :integer, description: "Instance HTTPS Port"},
+        start_priority: %Schema{type: :string, description: "Instance Start Priority"},
```

Comment:
The instance priority is a number with decimals

Reply:
Interesting, we're defining it as a string in commands, events and readmodels.

I'll keep it like this and create an item in the backlog if that's ok.

Reply:
Totally fine! We don't do anything more than displaying it, so string is even good for us